### DISCUSSION TITLE
issue: Tests suite not passing when node version lesser than 16 is used

### DISCUSSION THREAD

**Discussion by devakrishna33** [react-hook-form/react-hook-form#10247] [Q&A]
### Version Number

7

### Codesandbox/Expo snack

Sorry for not providing a link, I tried to change the node version on code sandbox to provide a link but not able to switch it back to older version.

### Steps to reproduce

1. Install NVM or other verion manager for node and install node version 14.21.3.
2. Run `pnpm install` and then run `pnpm test`, it will fail with the following error,

```
 FAIL  ../../src/__tests__/useForm/watch.test.tsx
  ● watch › should watch correctly with useFieldArray with action and then fallback to onChange

    TypeError: output.at is not a function

      309 |     render(<Component />);
      310 |
    > 311 |     expect(output.at(-1)).toEqual({
          |                   ^
      312 |       names: [],
      313 |     });
      314 |

      at at (src/__tests__/useForm/watch.test.tsx:311:19)
```

### Expected behaviour

This is happening because ".at" on arrays is not supported in node version <16. It will be great to have a note in CONTRIBUTING.md stating the minimum version of node required for developing is 16.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by bluebill1049** [comment]
could you share a GitHub repo or codesndbox? This is most likely due to setup. I think our requirement is node 12 plus only, also worth checking out our testing in the library as well.

**Comment by devakrishna33** [comment]
@bluebill1049 I have changed the node version to 14 and ran the tests with GitHub actions please check
 [here](https://github.com/devakrishna33/react-hook-form/actions/runs/4641943388/jobs/8215389330).
